_inc/client/security/monitor.jsx

```diff
@@ -35,9 +35,7 @@ export const Monitor = withModuleSettingsFormHelpers(
 						disableInDevMode
 						module={ this.props.getModule( 'monitor' ) }
 						support={ {
-							text: __(
-								'Keep tabs on your site and receive alerts the moment downtime is detected.'
-							),
+							text: __( 'We will continuously watch your site and receive alerts the moment downtime is detected.' ),
```

Comment:
Should it say "and you will"? Right now it sounds like "we" will receive the alerts.